hack/jenkins/job-configs/test-linkchecker.yaml

```diff
@@ -6,7 +6,9 @@
     logrotate:
         numToKeep: 200
     builders:
-        - shell: './hack/verify-linkcheck.sh'
+        - shell: |
+            export PATH=${PATH}:/usr/local/go/bin
```

Comment:
Nope, since it's not a job template it won't try to do variable substitution, I think.